Timer.R

```diff
@@ -0,0 +1,17 @@
+library(tictoc)
+
+# Time original method
+tic("Original method")
+source("scripts/original.R")
+toc()
+
+# Time parallel loop method
+tic("parallel loop method runtime")
+source("scripts/parallel_loop_method.R")
+toc()
+
+# Time parallel MTweedieTests method
+tic("Parallel MTweedieTests method runtime")
+source("scripts/parallel_mtweedietests_method.R")
+toc()
+ 
```

Comment:
Miss an interpretation of the results. The third one is supposed to be the quickest. The reason that the last one is quicker than the second one is because the second one parallelizes the rows in df. Because there are only 5 rows, having more than 5 cores will not make it quicker. The last one parallizes M, and can therefore utlize more cores in the computation